environment/api_app.tf

```diff
@@ -126,29 +126,29 @@ locals {
       { "name": "API_SECRETS_FRONT_KEY", "valueFrom": "${data.aws_secretsmanager_secret.front_api_client_secret.arn}" }
     ],
     "environment": [
-      { "name": "API_BEHAT_CONTROLLER_ENABLED", "value": "${local.test_enabled ? "true" : "false"}" },
-      { "name": "API_DATABASE_HOSTNAME", "value": "${aws_db_instance.api.address}" },
-      { "name": "API_DATABASE_NAME", "value": "${aws_db_instance.api.name}" },
-      { "name": "API_DATABASE_PORT", "value": "${aws_db_instance.api.port}" },
-      { "name": "API_DATABASE_USERNAME", "value": "digidepsmaster" },
+      { "name": "OPG_NGINX_SSL_FORCE_REDIRECT", "value": "1" },
       { "name": "API_REDIS_DSN", "value": "redis://${aws_route53_record.api_redis.fqdn}" },
-      { "name": "API_SECRETS_ADMIN_PERMISSIONS", "value": "[ROLE_ADMIN, ROLE_AD, ROLE_CASE_MANAGER]" },
-      { "name": "API_SECRETS_FRONT_PERMISSIONS", "value": "[ROLE_LAY_DEPUTY, ROLE_PA, ROLE_PROF, ROLE_PA_ADMIN, ROLE_PA_TEAM_MEMBER]" },
-      { "name": "API_SECURITY_ANONYMOUS", "value": "true" },
-      { "name": "NGINX_INDEX", "value": "app.php" },
+      { "name": "API_DATABASE_PORT", "value": "${aws_db_instance.api.port}" },
       { "name": "OPG_DOCKER_TAG", "value": "${var.OPG_DOCKER_TAG}" },
+      { "name": "API_SECRETS_ADMIN_PERMISSIONS", "value": "[ROLE_ADMIN, ROLE_AD, ROLE_CASE_MANAGER]" },
+      { "name": "API_DATABASE_HOSTNAME", "value": "${aws_db_instance.api.address}" },
+      { "name": "API_DATABASE_NAME", "value": "${aws_db_instance.api.name}" },
       { "name": "OPG_NGINX_CLIENTBODYTIMEOUT", "value": "240s" },
-      { "name": "OPG_NGINX_CLIENTMAXBODYSIZE", "value": "10M" },
-      { "name": "OPG_NGINX_INDEX", "value": "app.php" },
       { "name": "OPG_NGINX_ROOT", "value": "/app/web" },
       { "name": "OPG_NGINX_SERVER_NAMES", "value": "~.*" },
-      { "name": "OPG_NGINX_SSL_FORCE_REDIRECT", "value": "1" },
+      { "name": "OPG_NGINX_CLIENTMAXBODYSIZE", "value": "10M" },
+      { "name": "OPG_NGINX_INDEX", "value": "app.php" },
+      { "name": "API_BEHAT_CONTROLLER_ENABLED", "value": "${local.test_enabled ? "true" : "false"}" },
+      { "name": "API_SECURITY_ANONYMOUS", "value": "true" },
       { "name": "OPG_PHP_POOL_CHILDREN_MAX", "value": "12" },
-      { "name": "OPG_SERVICE", "value": "api" },
-      { "name": "OPG_STACKNAME", "value": "${terraform.workspace}" }
+      { "name": "API_SECRETS_FRONT_PERMISSIONS", "value": "[ROLE_LAY_DEPUTY, ROLE_PA, ROLE_PROF, ROLE_PA_ADMIN, ROLE_PA_TEAM_MEMBER]" },
+      { "name": "API_DATABASE_USERNAME", "value": "digidepsmaster" },
+      { "name": "OPG_STACKNAME", "value": "${terraform.workspace}" },
+      { "name": "NGINX_INDEX", "value": "app.php" },
+      { "name": "OPG_SERVICE", "value": "api" }
```

Comment:
What's the pattern to this new variable order?

Reply:
whatever causes the least diff - for some reason either the AWS API or terraform ends up setting them in this order